### ISSUE TITLE
installing RFAA using only PIP and avoiding conda and mamba

### ISSUE THREAD

**Issue by Sinsilcobio** [baker-laboratory/RoseTTAFold-All-Atom#119]
Hello,

Thanks for developing this nice tool!

I am trying to install it on a cluster that does not allow for the installation of packages through Conda and Mamba. 

I am trying to adjust the installation using a requirement.txt file to be utilized with Python and Pip. I have provided the content of the requirement file in the following and was able to use PIP to install all the packages in the file. Now I am stuck with installing the signal package.

Could you help me know how I could adjust my installation using pip to install the Signalp?

I already tried installing it by accessing the following GitHub repository. However, I can not find the directory "model_weights" directory in "RFAA/lib/python3.10/site-packages/signalp/"
$ pip install --no-deps git+https://github.com/fteufel/signalp-6.0.git

Your help is much appreciated!

contents of requiremets.txt file:

absl-py
aiohttp
aiosignal
asttokens
astunparse
async-timeout
attrs
blinker
brotli
#brotli-python
#bzip2
#c-ares
cached-property
cached_property
cachetools
certifi
cffi
charset-normalizer
click
colorama
contourpy
cryptography
cycler
deepdiff
dgl
executing
filelock
flatbuffers
fonttools
frozenlist
fsspec
gast
gmpy2
google-auth
google-auth-oauthlib
google-pasta
grpcio
h5py
icecream
idna
importlib-metadata
jinja2
joblib
keras
keras-preprocessing
kiwisolver
markdown
markupsafe
matplotlib
multidict
networkx
numpy
oauthlib
openbabel
opt_einsum
orjson
packaging
pandas
protobuf
psutil
pyasn1
pyasn1-modules
pycparser
pygments
pyjwt
pyopenssl
pyparsing
pysocks
python-dateutil
#python-flatbuffers
#python-tzdata
pytz
pyu2f
requests
requests-oauthlib
rsa
scikit-learn
scipy
setuptools
six
tensorboard
tensorboard-data-server
tensorboard-plugin-wit
tensorflow
tensorflow-estimator
termcolor
threadpoolctl
tk
toml
tomli
torch
torchaudio
torchvision
tornado
tqdm
typing-extensions
typing_extensions
unicodedata2
urllib3
werkzeug
wheel
wrapt
yarl
zipp

antlr4-python3-runtime
assertpy
configparser
docker-pycreds
e3nn
gitdb
gitpython
hydra-core
omegaconf
opt-einsum-fx
pathtools
promise
pynvml
pyrsistent
pyyaml
sentry-sdk
shortuuid
smmap
subprocess32
wandb

**Comment by pavelToman** [comment]
Hello!
I am either trying to install RoseTTA on cluster for UGent researches - we are using EasyBuild for this. I need to use pip too, not mamba or conda. 
@Sinsilcobio SignalP seems could be only optional dep - https://github.com/baker-laboratory/RoseTTAFold-All-Atom/issues/44
